prod-cockpit-rcar.yaml

```diff
@@ -63,6 +63,7 @@ common_data:
 
     # Add systemd configuration
     - [INIT_MANAGER, "systemd"]
+    - [DISTRO_FEATURES_append, " systemd"]
```

Comment:
As I remember you don't need to specify DISTRO_FEATURES systems, once you set INIT_MANAGER to "systemd".